### PR TITLE
Music: AI drums styling

### PR DESCRIPTION
**Pull request by breville**
This is a styling pass over the AI drums panel.

### before

Screenshot 2024-11-05 at 10 35 57 PM

### after

Screenshot 2024-11-05 at 10 34 11 PM

### video

https://github.com/user-attachments/assets/938cc7e9-dcd4-49be-bb2f-b763fdfe0916

### CODE REVIEW

**Comment by mikeharv** [comment]
Will these changes necessitate changes to panel levels?

**Comment by breville** [comment]
> Will these changes necessitate changes to panel levels?

Yes. We are planning on taking new captures anyway to use an updated background. This change is required first.

File: apps/src/music/views/PatternAiPanel.tsx [hunk 1]
```diff
@@ -17,7 +17,9 @@ const aiBotImages = [
 const arrowImage = require(`@cdo/static/music/music-callout-arrow.png`);
 
 import {Button} from '@cdo/apps/componentLibrary/button';
+import {SimpleDropdown} from '@cdo/apps/componentLibrary/dropdown';
 import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import Slider from '@cdo/apps/componentLibrary/slider/Slider';
```
Comment (sanchitmalhotra126) [inline apps/src/music/views/PatternAiPanel.tsx:28]: 🎉

**Review by sanchitmalhotra126 — APPROVED** [review]
Looks slick! Great work!

**Comment by dju90** [comment]
Looks amazing 🤩

**Pull request merged**